plat/allwinner/common/sunxi_common.c

```diff
@@ -10,8 +10,6 @@
 #include <sunxi_def.h>
 #include <xlat_tables_v2.h>
 
-#include "sunxi_private.h"
```

Comment:
The file defining the function should also include the header (to guarantee consistency in the prototype)